### ISSUE TITLE
Add support (or instruction) for pasting from clipboard

### ISSUE THREAD

**Issue by benjaminoakes** [rhysd/vim.wasm#27]
**Is your feature request related to a problem? Please describe.**

I couldn't paste some code I wanted to reformat.  Felt like vim.wasm could possibly do this, (mostly for convenience) on a Chromebook.  😄 

In particular, I wanted a simple reformat, which I knew I could do quickly in Vim:

In:

```
#Array
#Complex
#Float
#Hash
#Integer
#Rational
#String
```

Out:

```
(`Array()`, `Complex()`, `Float()`, `Hash()`, `Integer()`, `Rational()`, `String()`)
```

**Describe the solution you'd like**

Add support for pasting via Ctrl-V/Cmd-V, `"+p`, or another solution.  Add a short blurb somewhere on the page describing this.

**Additional context (if any)**

n/a

**Comment by rhysd** [comment]
Yeah, with tiny feature, Vim has its internal clipboard and it is not synchronized. I want to try this after enabling larger feature.

**Comment by rhysd** [comment]
After small investigation, synchronizing clipboard would not fit to this app because

- Clipboard API is not always available. Some browsers don't support it and a user may not permit to access to clipboard (read/write)
- Clipboard text may be sometimes big. In the case, sending the text to main thread at every yanking would be overhead. It requires UTF-8 to UTF-16 conversion and serialization for inter-threads communication (clipboard access is available on main thread only)

Considering these points, I want to implement clipboard support only for `"*` clipboard register.

- When user explicitly paste something from clipboard register such as `"*p`, it reads clipboard in main and Vim pastes the text in worker
- When user explicitly yank something to clipboard register such as `"*yy`, Vim sends the yanked text to main and main will write it to clipboard

Note that we need to consider the case where user does not permit access to clipboard. In the case this feature is simply disabled.

**Comment by rhysd** [comment]
This was implemented at 7de8e98.

> Clipboard register `"*` is supported.  For example, paste system clipboard text to Vim with `"*p` or `:put *`, and copy text in Vim to system clipboard with `"*y` or `:yank *`.  If you want to synchronize Vim's clipboard with system clipboard, `:set clipboard=unnamed` should work like normal Vim.

![tmp](https://user-images.githubusercontent.com/823277/59963693-a7a42780-9531-11e9-9ffc-6e35c6c96ec5.gif)
